runtime/src/bank.rs

```diff
@@ -271,23 +211,15 @@ pub struct Bank {
     hard_forks: Arc<RwLock<HardForks>>,
 
     /// The number of transactions processed without error
-    #[serde(serialize_with = "serialize_atomicu64")]
-    #[serde(deserialize_with = "deserialize_atomicu64")]
```

Comment:
Using the serde-provided Atomic Serialize/Deserialize is good thing! However I think they're logically not coupled with this snapshot versioning pr. Could you create a separate pr to remove these and corresponding code in `serde_utils.rs`? I'll quickly approve them. :)

Reply:
PR #10096 deletes those functions.